sympy/concrete/summations.py

```diff
@@ -254,6 +254,28 @@ def _eval_derivative(self, x):
         else:
             return NotImplementedError('Lower and upper bound expected.')
 
+    def _eval_integral(self, x):
+        if isinstance(x, Symbol) and x not in self.free_symbols:
+            return x*self
```

Comment:
This case is just an extra method in the Class Sum just like "_eval_differentiation" which firsts differentiates the variable and then returns Sum class containing this new differentiated variable ,this method (_eval_integral) first integrates the variable and then return the Sum of integrated variable with the same limits of summation . I think integrate was not able to handle this case (that is first integrating and then summing)